### ISSUE TITLE
phpcs.enable option not working?

### ISSUE THREAD

**Issue by 9brada6** [ikappas/vscode-phpcs#155]
Hi... for some reasons, even if I put phpcs.enable = false setting in user or workspace directory, the phpcs still show the error that it can't find the executable... Is this supposed to be like this?

![image](https://user-images.githubusercontent.com/19844284/65678632-5178e000-e05c-11e9-80de-4769d5d70056.png)

**Comment by simondubois** [comment]
@9brada6 I face the same issue. Phpcs is disabled in the workspace but the plugin still tries to run. Could you share how you fixed it?

**Comment by 9brada6** [comment]
> @9brada6 I face the same issue. Phpcs is disabled in the workspace but the plugin still tries to run. Could you share how you fixed it?

Hi... yes, I just disabled the plugin globally and enabled it for the workspace :)

**Comment by simondubois** [comment]
Thanks for the info.
I switched to https://github.com/wongjn/vscode-php-sniffer to solve the problem.

**Comment by 9brada6** [comment]
Ah cool, in fact I switched too, to [valeryanm.vscode-phpsab](https://github.com/valeryan/vscode-phpsab.git). I like phpsab more because like phpcs it auto-detect the code snipper executable, which is nice to not have to write everytime manually, and has also auto-format(if the error can be fixed, like auto space). 

The only problem with valeryanm.vscode-phpsab is that somehow the problems shown will not go away if the file was open in preview mode(and then closed). This is not a problem for me since I have set anyway to show only the problems in the current file displayed, and to open a file in
preview mode is anyway disabled.

So: 
- phpcs: enable option don't work, 
- wongjn/phpcs: you will need to insert the executable manually.
- valeryanm/phpsab: the problems will not go away if you open a file in preview mode and close it.

Pick your poison.
